src/pages/STG_PanelAddrVerification.page

```diff
@@ -2,19 +2,19 @@
 
     <script>
         var j$ = jQuery.noConflict();
```

Comment:
On all of our settings pages, the Label font is a different size than the value font, and so their baseline is not lining up.  It looks off.  Worse, those properties who are checkboxes look very off, as the checkbox is way above the baseline.  (I just put this comment on the first page)